src/config.ts

```diff
@@ -35,10 +41,21 @@ export const CONFIG_HELP_SECTION = toHelpSection(
 
 export abstract class ConfigCommand<T> extends SfCommand<T> {
   protected responses: ConfigResponses = [];
-
+  // eslint-disable-next-line class-methods-use-this
+  public calculateSuggestion(cmd: string): string {
```

Comment:
Am I mistaken, but this is being used to get suggestions for config keys. Suggest changing `cmd` to something more fitting.